public/angular/userManagement/userManagement.module.js

```diff
@@ -69,6 +74,37 @@ angular.module('tripleT.userManagement', ['ngRoute', 'ngMessages'])
     }
 })
 
+.controller('ChangePwdCtrl', 
+  function($scope, $http, $timeout) {
+    // userInput = {
+    //    password: their current password
+    //    newPassword: what they want it to be
+    //    confirmNewPassword: above. hopefully.   
+    //  }
+    $scope.changePassword = function(userInput) {
+      $scope.passwordChanged = false;
+      $scope.errorMsg = null;
```

Comment:
Because we want to make this a modal in the future, not sure if the $scope.errorMsg will override another errorMsg in some inherited scope. I don't think it'll be a problem, but be on the lookout.